release-notes/opendistro-for-elasticsearch-asynchronous-search.release-notes-1.13.0.0.md

```diff
@@ -0,0 +1,12 @@
+## 2021-02-03 Version 1.13.0.0
+Initial launch of Asynchronous Search on opendistro-for-elasticsearch
+
+### Maintenance
+* Supports Elasticsearch 7.10.2 ([#32](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/32))
+
+
+### Features
+* Listener framework for asynchronous search ([#2](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/2))
+* Service layer and Transport Handlers ([#8](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/8))
+* Rest Layer and Async Search Cleanup Management ([#9](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/9))
+* Integrates security with asynchronous search ([#11](https://github.com/opendistro-for-elasticsearch/asynchronous-search/pull/11))
```

Comment:
I guess we should talk about the Cx and usage of the APIs instead

Reply:
added a brief intro about plugin.